android/library/src/main/java/io/logto/android/callback/AuthenticationCallback.kt

```diff
@@ -0,0 +1,5 @@
+package io.logto.android.callback
+
+import io.logto.android.model.Credential
+
+interface AuthenticationCallback : Callback<Credential>
```

Comment:
i think  coroutines + lambda functions with `try - catch` blocks will be more clear instead of callbacks.. we can discuss it

Reply:
TBD: [LOG-84](https://linear.app/silverhand/issue/LOG-84/use-coroutines-in-project)